web/packages/shared/components/UnifiedResources/CardsView/CardsView.tsx

```diff
@@ -56,6 +57,7 @@ export function CardsView({
           selected={selectedResources.includes(key)}
           selectResource={() => onSelectResource(key)}
           pinResource={() => onPinResource(key)}
+          onCopyToClipboard={onCopyToClipboard}
```

Comment:
Overall I think changing the permissions is a better way forward, but if we were to stick with this, I'd rather avoid using the `on` suffix here. It sounds like we're adding some extra actions that's supposed to happen when someone uses an existing interaction (like `onClick`, `onHover` etc.), but what we're actually doing is just passing a different implementation of a function that's supposed to copy something to the clipboard. So I feel like just `copyToClipboard` would be sufficient here.